### DISCUSSION TITLE
Wrong encoding in the custom property (russian symbols)

### DISCUSSION THREAD

**Discussion by XIOLog** [spatie/laravel-medialibrary#2736] [Q&A]
Hi. I have a trouble. 

I use ```"spatie/laravel-medialibrary": "^9.7"```

I have an array $image.

![Image Array](https://i.imgur.com/eW3ldO5.png)

There is nothing wrong with this array.

But when I save the image to the media table, I get the wrong save data ($image['title']) in "custom_properties"

```php
$model
    ->addMediaFromDisk($image['image'], 'livewire-tmp')
    ->withCustomProperties(['title' => $image['title'], 'position' => $image['position']])
```

![media table](https://i.imgur.com/NVEoSCt.png).

I couldn't find anything in the config.

**Comment by vaiil** [comment]
@XIOLog it seems that it is just encoded JSON (array with custom properties is encoded using json_encode function)

**Reply by freekmurze** [reply]
Feel free to PR improvements. Make sure to include tests.
